be/src/types/hll.h

```diff
@@ -135,7 +135,7 @@ class HyperLogLog {
     HllDataType _type = HLL_DATA_EMPTY;
 
     using ElementSet = phmap::flat_hash_set<uint64_t>;
-    std::unique_ptr<ElementSet> _hash_set;
+    ElementSet _hash_set;
```

Comment:
should add comment for this optimization

Reply:
Thanks, fixed.